index.html

```diff
@@ -138,12 +148,7 @@ <h1>About</h1>
         </p>
         <button type="button" class="button">
           Get my CV
-          <img
-          src="./assets/img/ic_download.png"
-          width="32"
-          alt="Download icon"
-          class="dl-icon"
-          />
+          <img src="./assets/img/ic_download.png" width="32" alt="Download icon" class="dl-icon" />
         </button>
       </div>
       <ul class="main-list">
```

Comment:
- [ ] Kindly update this section to make it match Figma design 💯 suggestion: you can use a grid that has 2 rows and 2 columns 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/187006121-aa058634-125a-4239-b9fd-73bb7dc8cbc6.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/187006138-78901dfe-2b13-400c-9c68-cd005074689d.png)